hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/TestMetadataTableSupport.java

```diff
@@ -81,15 +81,17 @@ void testRecreateMDTForInsertOverwriteTableOperation() {
           .setBasePath(mdtBasePath).build();
       HoodieActiveTimeline timeline = mdtMetaClient.getActiveTimeline();
       List<HoodieInstant> instants = timeline.getInstants();
-      assertEquals(4, instants.size());
+      assertEquals(5, instants.size());
```

Comment:
i'm thinking if we should make it more dynamic based on MetadataPartitionType.getEnabledPartitions. We always hit this when we enable any index by default. Something to followup.

Reply:
ack